src/database/queries/auditLogs/insertOne.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO audit_logs_2 (
+	user_keycloak_user_id,
```

Comment:
Let's also store the administrator status of the operating agent.

Reply:
It is being stored in the `queryParameters` along with the user ID. The reason for storing the (duplicative) user ID separately is the anticipation that we will want to be able to say "show me X user's actions." Do you think we'll have a commonly called query of "show me all administrative actions respective of users"?

Example row from the view in the original PR:
` 2025-04-16 15:17:30.980584-05 | 8f6e6dd9-d4af-45db-af50-712f7e962cd7 | 14554 |     2 | Called query applicationForms.selectWithPagination  | {"limit": 10, "offset": 0, "authContextKeycloakUserId": "8f6e6dd9-d4af-45db-af50-712f7e962cd7", "authContextIsAdministrator": true}`

Reply:
Added the extra column `user_is_administrator` to the table.